### PR TITLE
Add share widget to Hello product page. Bug 1119849.

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

**Comment by jpetto** [comment]
May want to test on demo prior to merging, but good for review.

**Comment by jpetto** [comment]
Verified on demo5. Good to merge after review.

**Comment by alexgibson** [comment]
Not related to this PR, but there should be a space between attributes on this line in the share CTA: https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/macros.html#L361

File: bedrock/firefox/templates/firefox/hello/index.html [hunk 1]
```diff
@@ -25,15 +27,28 @@
...
       {% block site_header_logo %}
-        {{ super() }}
-      {% endblock %}
+        <div id="hello-logo-wrapper">
+          {{ super() }}
+
+          {%
+          set share_urls = {
+            'twitter': 'http://mzl.la/1wz83AN',
+            'googleplus': 'https://www.mozilla.org/' + LANG + '/firefox/hello/?utm_source=Google&utm_medium=social&utm_campaign=Hello&utm_content=WebsiteShare',
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/hello/index.html]: As Twitter uses a short URL above, I'm assuming we're sharing a non-locale specific URL there - but here a locale specific one? Should they be the same?

File: media/css/firefox/hello/index.less [hunk 2]
```diff
@@ -100,60 +114,59 @@ main {
...
-        }
+    &:hover,
+    &:focus {
+        background: transparent;
+    }
 
-        &.open > div > a{
-            background-position: 0 8px;
+    h3 {
+        .font-size(12px);
```
Comment (alexgibson) [inline media/css/firefox/hello/index.less]: Nit: `@smallFontSize`

File: bedrock/firefox/templates/firefox/hello/index.html [hunk 3]
```diff
@@ -25,15 +27,28 @@
...
+            'facebook': 'https://www.mozilla.org/' + LANG + '/firefox/hello/?utm_source=Facebook&utm_medium=social&utm_campaign=Hello&utm_content=WebsiteShare'
+          }
+          %}
+          {{ share_cta(_('Share'), share_urls, _('Meet Firefox Hello, the easiest way to connect for free over video with anyone, anywhere.'), 'spread-hello') }}
+          {% endblock %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/hello/index.html]: Should the `{% endblock %}` here come after the `</div>` below?

**Comment by alexgibson** [comment]
Couple of nits aside, this looks good to me r+wc :+1:

File: bedrock/firefox/templates/firefox/hello/index.html (hunk 1 above)
Comment (jpetto) [inline bedrock/firefox/templates/firefox/hello/index.html]: Hmm, it is a good question. I followed suit from the Privacy Day widget. My thought was that we'd be saving users a redirect, but I guess that would be at the cost of possibly linking to an incorrect locale (depending on who clicks the shared link).

After a bit more thought, I think I'd rather leave the locale out.

File: bedrock/firefox/templates/firefox/hello/index.html (hunk 3 above)
Comment (jpetto) [inline bedrock/firefox/templates/firefox/hello/index.html]: Ah, it should indeed. Good catch!

**Comment by jpetto** [comment]
Think I got all the corrections covered. Thanks @alexgibson!

**Pull request merged**